docs/FAQ.md

```diff
@@ -73,7 +73,6 @@ We typically do not assign issues to anyone other than long-time contributors. I
 
 Our community moderators are our heroes. Their voluntary contributions make
 freeCodeCamp a safe and welcoming community.
```

Comment:
Sorry for the confusion, my dyslexia triggered, and I kept misusing a label.

The issue are the line breaks no extra line, notes how sentences in the markdown file in the editor, are broken into multiple lines, although they are a one line. Like shown below in the suggestion. I need the line breaks to be removed for easier translation

```suggestion
Our community moderators are our heroes. Their voluntary contributions make freeCodeCamp a safe and welcoming community.

```

Reply:
No worries man i'll get right on it

Reply:
so do you want the changes in just these two lines or the entire moderator section?

Reply:
so i have removed all the unnecessary line breaks